docs/README.md

```diff
@@ -85,6 +85,12 @@ This app exports all of the following blocks. You can see more detailed informat
   },
   "summary-coupon": {
     // this block doesn't have any props
+  },
+  "summary-installments": {
+    "props": {
+      "message": "",
```

Comment:
Can we change to a valid example?

Reply:
@klzns Sorry, I forgot to add the workspace link. Here it is: https://paymentdata--tokstokio.myvtex.com/venda-corporativa
Also, just added a valid example to README.